_posts/2025-09-15-quarkus-a2a-multi-agent-content-creation.adoc

```diff
@@ -0,0 +1,244 @@
+---
+layout: post
+title: 'Multi-Language Agent Collaboration and Interoperability with A2A'
+date: 2025-09-15
+tags: ai a2a
+synopsis: Let's learn how to build a multi-agent system where agents written in different languages and with different LLM frameworks can seamlessly collaborate using A2A.
+author: fjuma
+---
+:imagesdir: /assets/images/posts/quarkus-a2a-multi-agent-content-creation
+
+Building a multi-agent system can involve using different languages to meet specific needs. The https://a2a-protocol.org/latest/[Agent2Agent (A2A) protocol] is an open standard that enables AI agents to communicate and collaborate with one another, regardless of each agent's underlying technology stack.
+
+In this post, we'll see how to create a multi-agent system, where agents written in Java, Python, and TypeScript work together to accomplish a goal: content creation. The multi-agent system uses A2A for communication between the AI agents.
```

Comment:
Out of scope for this post, but at some point, we need to find another use case (NB: I'm using the same too)

Reply:
+1, coming up with additional use cases is something we've been discussing.